src/AcceptanceTesting/SpecByExample/SFA.DAS.Payments.AcceptanceTests/Features/Ilr Changes/learner_changes_contract_type.feature

```diff
@@ -0,0 +1,75 @@
+﻿@learner_changes_contract_type
+Feature: Learner changes contract type
+
+Scenario: DPP-965_01 - Levy apprentice, provider edits contract type (ACT) in the ILR, previous on-programme and English/math payments are refunded and repaid according to latest contract type
```

Comment:
Specflow converts "DPP-965_01" in the generate code behind file to "DPP_965_01", which means if you search for "DPP-965_01" in the Unit Test Explorer, the test isn't found.
A minor issue, but we might want the scenario name and generated test name to match.